llama_hub/tools/metaphor/base.py

```diff
@@ -0,0 +1,100 @@
+"""Metaphor tool spec."""
+
+from llama_index.tools.tool_spec.base import BaseToolSpec
+from typing import Optional, List
+from llama_index.readers.schema.base import Document
+import requests
+import urllib.parse
+import datetime
+
+class MetaphorToolSpec(BaseToolSpec):
+    """Metaphor tool spec."""
+
+    spec_functions = ["metaphor_search", "retrieve_documents", "current_date", "find_similar"]
+
+    def __init__(self, api_key: str) -> None:
+        """Initialize with parameters."""
+        from metaphor_python import Metaphor
+        self.client = Metaphor(api_key=api_key)
+
+    def metaphor_search(
```

Comment:
What's the reasoning behind prefixing `metaphor` here but not in the other functions? I assumed this is to help the agent better understand the tool by giving it a less generic name, but should that then have consistency across the 3 functions?

Reply:
good point, I think that just `search` is fine 👍